src/GraphQl/Type/SchemaBuilder.php

```diff
@@ -60,7 +60,7 @@ public function getSchema(): Schema
             /** @var array<string, mixed> $graphqlConfiguration */
             $graphqlConfiguration = $resourceMetadata->getGraphql() ?? [];
             foreach ($graphqlConfiguration as $operationName => $value) {
-                if ('query' === $operationName) {
+                if ('item_query' === $operationName || 'collection_query' === $operationName) {
```

Comment:
Please separate into two conditions and change the value of `$itemConfiguration` and `$collectionConfiguration` to false in the `getQueryFields` call.